dataprofiler/profilers/profiler_options.py

```diff
@@ -695,14 +695,15 @@ def _validate_helper(self, variable_path='OrderOptions'):
 
 class CategoricalOptions(BaseInspectorOptions):
 
-    def __init__(self):
+    def __init__(self, is_enabled=True, top_k_categories=None):
         """
         Options for the Categorical Column
 
         :ivar is_enabled: boolean option to enable/disable the column.
         :vartype is_enabled: bool
         """
-        BaseInspectorOptions.__init__(self)
+        BaseInspectorOptions.__init__(self, is_enabled=True)
```

Comment:
You take `is_enabled` in as a parameter with a default of `True`, but then just pass `True` anyway, meaning that the user could never have it be `False` via construction, make sure to pass the `is_enabled` given by the user (optionally) here instead.